meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainerAuto.sh

```
@@ -11,103 +31,37 @@ killall  -9 jsdelivr-systemWatchdog.sh
-
-
-if [ "$SERVER_VER" = "$CURRENT_VER" ]; then
-    echo "The version are the same... aborting upgrade and waiting for the next upgrade cycle" > /dev/tty5
-    echo "Resuming the MANDATORY reboot script" > /dev/tty5
-    killall -CONT jsdelivr-mandatoryReboot.sh
-    killall -CONT jsdelivr-systemMonitor.sh
-    killall jsdelivr-keepWatchdogHappy.sh
-    exit 2
-fi
+CONTAINERS=$(docker ps -aq 2>/dev/null)
+[ -n "$CONTAINERS" ] && docker stop $CONTAINERS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)